### ISSUE TITLE
Delayed ability to walk with support

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5884]
**Preferred term label:** Delayed ability to walk with support

**Synonyms** Delayed ability to cruise

**Definition (free text, please give PubMed ID)** A failure to achieve the ability to walk with support (cruise) at an appropriate developmental stage. 

**Parent term (use hpo.jax.org/app)** Delayed ability to walk HP:0031936

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
